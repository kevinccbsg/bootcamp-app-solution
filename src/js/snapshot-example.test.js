const { beerTemplate, buttonTemplate } = require('./beers');

describe('Snapshot', () => {
  it('beerTemplate Snapshot', () => {
    const tree = beerTemplate({
      beerId: 2,
      image: 'image',
      name: 'name 1',
      description: 'description 1',
      contributedBy: 'kevin',
      firstBrewed: '2019-12',
      likes: 2,
    });
    expect(tree).toMatchSnapshot();
  });

  it('buttonTemplate Snapshot', () => {
    const tree = buttonTemplate(2);
    expect(tree).toMatchSnapshot();
  });
});