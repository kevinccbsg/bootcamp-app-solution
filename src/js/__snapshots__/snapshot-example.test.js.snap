// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot beerTemplate Snapshot 1`] = `
"
  <a href=\\"detail.html?id=2\\">
    <div id=2 class=\\"card\\">
      <div class=\\"card-image-container\\">
        <picture>
          <img class=\\"card-image\\" src=image alt=name 1 />
        </picture>
      </div>
      <div class=\\"card-content\\">
        <div>
          <h2>name 1</h2>
        </div>
        <p>description 1</p>
      </div>
      <div class=\\"card-footer\\">
        <div>
          <div>kevin</div>
          <div>2019-12</div>
        </div>
        
  <button>
    <i class=\\"fas fa-heart\\"></i>
    2
  </button>

      </div>
    </div>
  </a>
"
`;

exports[`Snapshot buttonTemplate Snapshot 1`] = `
"
  <button>
    <i class=\\"fas fa-heart\\"></i>
    2
  </button>
"
`;
